remediation/self-node-remediation/known-issues.md

```diff
@@ -14,4 +14,5 @@ Self Node Remediation has several known issues:
 4. Upon installing self node remediation operator, it might take up to 2 minutes before the daemonset is deployed
 5. IPv6 is not supported
 6. At least two workers are required to get node fencing
-7. At release 0.3 channel name was changed from "**alpha**" to "**stable**". This means that when releases prior to 0.3 are upgraded, the channel name needs to be changed manually 
+7. At release 0.3 channel name was changed from "**alpha**" to "**stable**". This means that when releases prior to 0.3 are upgraded, the channel name needs to be changed manually
+8. When installed on a k8s 1.25+ cluster a privileged psa label (i.e. *pod-security.kubernetes.io/enforce: privileged*) must be set manually on the namespace snr will be installed in. This is because SNR agents require privileged permissions in order to reboot the node (in case it needs to be remediated). More info can be found [here](https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/).
```

Comment:
This paragraph seems a bit complex to read. What about something like the following?

> When installed on a k8s 1.25+ cluster, you must manually set a privileged PSA label (i.e. *pod-security.kubernetes.io/enforce: privileged*) on the same SNR's namespace to give SNR agents permissions to reboot the node (in case it needs to be remediated).

Reply:
Hmm How about this ? :
On a k8s 1.25+ cluster prior to installing SNR,  you must manually set a privileged PSA labe (i.e. pod-security.kubernetes.io/enforce: privileged) on SNR's namespace to give SNR's agents permissions to reboot the node (in case it needs to be remediated).

Reply:
Or 

> Prior to installing SNR on a Kubernetes 1.25+ cluster, a user must manually set a privileged PSA labe (i.e. pod-security.kubernetes.io/enforce: privileged) on SNR's namespace. It gives SNR's agents permissions to reboot the node (in case it needs to be remediated).

Reply:
I like this  version the best :slightly_smiling_face: .
 @clobrano is that ok by you ?

Reply:
perfect :ok_hand: